core/src/components/radio/radio.tsx

```diff
@@ -135,7 +135,7 @@ export class Radio implements ComponentInterface {
     ev.stopPropagation();
     ev.preventDefault();
 
-    this.el.focus();
+    this.nativeInput.focus();
```

Comment:
It looks like this causes some issues with the legacy syntax just in Safari. Doing a Shift+Tab to get back to the previous radio group requires pressing Shift+Tab twice.
```suggestion
    const element = this.legacyFormController.hasLegacyControl() ? this.el : this.nativeInput;
    element.focus();
```

Let's keep focusing `this.el` only when in the legacy syntax.